### PR TITLE
Added DFN-8-1EP_4x4mm_P0.8mm_EP2.3x3.24mm

### PR DESCRIPTION
**Pull request by JeppeSRC**
[Datasheet](https://www.st.com/resource/en/datasheet/ld1086.pdf)

### CODE REVIEW

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml [hunk 1]
```diff
@@ -297,6 +297,46 @@ DFN-8-1EP_3x3mm_P0.65mm_EP1.7x2.05mm:
...
+    minimum: 0.23
+    nominal: 0.30
+    maximum: 0.38
+  lead_len:
+    minumum: 0.4
+    nominal: 0.5
+    maximum: 0.6
+
+  EP_size_x: 2.30
+  EP_size_y: 3.24
```
Comment (poeschlr) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml]: Enter size of the package ep with this parameter and use EP_size_x_overwrite to force it to use the pad size as defined in the datasheets suggested footprint. (same with y size)

Comment (JeppeSRC) [reply scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml]: So I made the changes but the EP didn't get overridden. Or did I fuck up?

Comment (poeschlr) [reply scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml]: You have a typo in the parameter name.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml [hunk 2]
```diff
@@ -321,8 +321,19 @@ DFN-8-1EP_4x4mm_P0.8mm_EP2.3x3.24mm:
...
+    minimum: 2.05
+    nominal: 2.20
+    maximum: 2.30
+
+  EP_size_y:
+    minimum: 2.82
+    nominal: 3
+    maximum: 3.23
+
+  EP_size_x_override: 2.30
```
Comment (poeschlr) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml]: `EP_size_x_override` should be `EP_size_x_overwrite`

Comment (JeppeSRC) [reply scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml]: Oh, that explains it. 🤦‍♂

**Comment by poeschlr** [comment]
thanks

**Pull request merged**